apps/frontend/src/components/BlogNav.tsx

```
@@ -0,0 +1,95 @@
+                    <span>Search Blog</span>
+                </label>
+
+                <div className="transition-layout absolute left-0 top-12 flex h-lvh max-h-0 w-full flex-col items-center overflow-hidden bg-white duration-500 ease-in peer-checked/search:max-h-lvh peer-checked/search:pt-8">
+                    <form
+                        id="search"
+                        onSubmit={(event) => handleSubmit(event)}
+                        className="transition-text w-full max-w-[1152px] px-4 text-2xl"
+                    >
+                        <input
+                            ref={input}
+                            autoFocus
```

Biome (1.9.4)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)